src/components/App/App.js

```diff
@@ -47,8 +47,7 @@ class App extends Component {
       return acc;
     }, this.state.components);
 
-    this.props.store.data = window.data;
-    this.props.store.colors = window.colors;
+    this.props.store.addComponents(components);
```

Comment:
It's cleaner to not changing the MobX store value outside of it and also because data and color never have to change, I remove it from the store.